elements/pl-dropdown/pl-dropdown.py

```diff
@@ -0,0 +1,174 @@
+import prairielearn as pl
+import lxml.html
+import random
+import chevron
+from enum import Enum
+
+WEIGHT_DEFAULT = 1
+BLANK_ANSWER = ' '
```

Comment:
Should we allow for customization of this? (Similar to `pl-*-input` placeholder text?) Not 100% sure on that.

If anything, let's prefix with `_DEFAULT`

Reply:
I would say I am on the fence here too. I thought about it too, but it seems unnecessary. It does not seem to add value unless people using this feature would be confused by a blank entry.

One problem is that once the blank entry appears in the drop-down list, it is always a submittable option without adding more complexity with JavaScript that stops submitting the form if a blank answer is selected. It may be nice to have some placeholder text instead of blank text, but I don't see the value in customizing it. 

Can you think of an ideal default selection to fill it with? ie. "Select answer"? Or maybe we should preface questions with "Select answer" outside of the scope of the drop-down element?

Reply:
I agree largely. Even as I was typing it, I'm not sure what the benefit would be if the blank was switched to "Select answer:" as it would already be picked up as an issue on submission.

Let's skip implementing the option unless it's requested by others.